payments-core/src/main/java/com/stripe/android/view/AuthActivityStarter.kt

```diff
@@ -1,5 +1,8 @@
 package com.stripe.android.view
 
-internal fun interface AuthActivityStarter<ArgsType> {
+import androidx.annotation.RestrictTo
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+fun interface AuthActivityStarter<ArgsType> {
```

Comment:
Audit

Reply:
This class needs to be fully public in order for `WeChatAuthStarter` to extend